packages/core/src/application.ts

```diff
@@ -241,3 +328,22 @@ export interface ApplicationConfig {
 
 // tslint:disable-next-line:no-any
 export type ControllerClass = Constructor<any>;
+
+/**
+ * A Booter class interface
+ */
+export interface Booter {
+  config?(): void;
```

Comment:
Should we use verbs for the phases/methods? For example, `configure`.

Reply:
+1 for using verbs for phases/method names, that's the usual convention AFAIK.